dubbo-common/src/main/java/org/apache/dubbo/common/URL.java

```diff
@@ -1298,7 +1298,7 @@ public String getPathKey() {
     }
 
     public static String buildKey(String path, String group, String version) {
-        return BaseServiceMetadata.buildServiceKey(path, group, version);
+        return BaseServiceMetadata.buildServiceKey(path, group, version).intern();

```

Comment:
use intern() in BaseServiceMetadata.buildServiceKey?

Reply:
done